src/main/java/iudx/catalogue/server/apiserver/util/Constants.java

```diff
@@ -38,20 +38,20 @@ public class Constants {
   /**
    *Routes
    */
-  private static String basePath = "/iudx/cat/v1";
+  // private static String basePath = "/iudx/cat/v1";
```

Comment:
delete the variable instead of commenting it out

Reply:
I've deleted the variable [here](https://github.com/datakaveri/iudx-catalogue-server/pull/196/commits/51929c71a7eeeff73c22dad4bbe5ff5af8c52c5d)